src/test/crux-wallet-client.ts

```diff
@@ -161,7 +182,25 @@ describe('CruxWalletClient Tests', function() {
             const fetchedAddressMap2 = await this.cc.getAddressMap();
             expect(fetchedAddressMap2['bitcoin']['addressHash']).equals(newAddressMap['bitcoin']['addressHash']);
         });
-    });
 
+        it('getAssetMap Check', async function() {
+            const resolvedClientAssetMapping = {
+                "bitcoin" :             {
```

Comment:
can't we refer it from existing define location then copying the JSON at multiple places?

Reply:
It was being used only once, moved it to top